.commitlintrc.json

```
@@ -18,6 +18,7 @@
     }
   },
   "rules": {
-    "scope-enum": [2, "always", ["admin", "profile", "pwdreset", "consolidator", "linker", "publications", "openapi", "lib", "npm", "deps", "deps-dev"]]
+    "scope-enum": [2, "always", ["admin", "profile", "pwdreset", "consolidator", "linker", "publications", "openapi", "lib", "npm", "deps", "deps-dev"]],
+    "body-max-line-length": [0, "always", 100]
```

Forgot the change itself ?
```suggestion
    "body-max-line-length": [0, "always", 150]
```

---

Sorry, this was just testing of these rules. I changed it to 200 characters and I changed the first number to `2` (meaning of this number is the level: `0 disables the rule, 1 will be considered a warning, 2 as an error`)